src/diffpy/utils/resampler.py

```diff
@@ -15,52 +15,50 @@
 
 """Various utilities related to data parsing and manipulation."""
 
-import numpy
+import numpy as np
 
 
-# NOTE - this should be faster than resample below and conforms more closely to
-# numpy.interp. I'm keeping resample for legacy reasons.
 def wsinterp(x, xp, fp, left=None, right=None):
```

Comment:
I was reading the code again, but I thought `wsinterp` docstrings could be improved by making it explicit.